src/pages/Facility/services/pharmacy/DispensesView.tsx

```diff
@@ -99,63 +76,35 @@ export default function DispensesView({
           <PatientHeader patient={patientData} facilityId={facilityId} />
         </Card>
       )}
-      <Tabs
+      <FilterTabs
         value={status}
         onValueChange={(value) =>
           navigate(
             `/facility/${facilityId}/locations/${locationId}/medication_dispense/patient/${patientId}/${value}`,
           )
         }
-      >
-        <TabsList className="w-full justify-evenly sm:justify-start border-b rounded-none bg-transparent p-0 h-auto overflow-x-auto">
-          {visibleTabs.map((statusValue) => (
-            <TabsTrigger
-              key={statusValue}
-              value={statusValue}
-              className="border-b-3 px-1.5 sm:px-2.5 py-2 text-gray-600 font-semibold hover:text-gray-900 data-[state=active]:border-b-primary-700  data-[state=active]:text-primary-800 data-[state=active]:bg-transparent data-[state=active]:shadow-none rounded-none"
-            >
-              {t(statusValue)}
-            </TabsTrigger>
-          ))}
-          {dropdownItems.length > 0 && (
-            <DropdownMenu>
-              <DropdownMenuTrigger asChild>
-                <Button
-                  variant="ghost"
-                  className="text-gray-500 font-semibold hover:text-gray-900 hover:bg-transparent pb-2.5 px-2.5"
-                >
-                  {t("more")}
-                  <ChevronDown />
-                </Button>
-              </DropdownMenuTrigger>
-              <DropdownMenuContent align="end">
-                {dropdownItems.map((statusValue) => (
-                  <DropdownMenuItem
-                    key={statusValue}
-                    onClick={() => handleDropdownSelect(statusValue)}
-                    className="text-gray-950 font-medium text-sm"
-                  >
-                    {t(statusValue)}
-                  </DropdownMenuItem>
-                ))}
-              </DropdownMenuContent>
-            </DropdownMenu>
-          )}
-        </TabsList>
+        options={tabOptions}
+        variant="underline"
+        showAllOption={false}
+        showMoreDropdown={true}
+        maxVisibleTabs={4}
+        defaultVisibleOptions={defaultVisibleStatuses}
+      />
```

Comment:
nav-tabs is the right one to be used here instead of filter-tabs. this is not performing any filtering. instead it's performing a page navigate.